### PR TITLE
Refactor the readme a bit and add more screenshots

### PR DESCRIPTION
**Pull request by GeorgianaElena**
This simplifies the language in some places, re-orders some sections, adds a toctree and more screenshots.

Preview here: https://github.com/GeorgianaElena/hub-user-image-template/blob/screenshots/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,90 @@
...
+### The environment
 
-* It provides an example of a `environment.yml` conda configuration file.
-  This file can be used to list all the conda packages that need to be installed by `repo2docker` in 
-  your environment.
-  The `repo2docker-action` will update the [base repo2docker](https://github.com/jupyterhub/repo2docker/blob/HEAD/repo2docker/buildpacks/conda/environment.yml)
-  conda environment with the packages listed in this `environment.yml` file.
+It provides an example of a `environment.yml` conda configuration file for repo2docker to use.
+This file can be used to list all the conda packages that need to be installed by `repo2docker` in your environment.
+The `repo2docker-action` will update the [base repo2docker](https://github.com/jupyterhub/repo2docker/blob/HEAD/repo2docker/buildpacks/conda/environment.yml) conda environment with the packages listed in this `environment.yml`
```
Comment (damianavila) [inline README.md]:
```suggestion
The `repo2docker-action` will update the [base repo2docker](https://github.com/jupyterhub/repo2docker/blob/HEAD/repo2docker/buildpacks/conda/environment.yml) conda environment with the packages listed in this `environment.yml` file.
```

File: README.md [hunk 2]
```diff
@@ -1,35 +1,90 @@
...
+file.
```
Comment (damianavila) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -41,9 +96,9 @@ When you have completed these steps, you should have:
 
   ![Secrets](images/secrets.png)
 
-### 3. Update workflow config to push the image to quay.io
+### 3. Enable image push to quay.io
 
-Step 2 should have provided the appropriate credentials to push the image to quay.io. But the `IMAGE_NAME` option also has to be configured for the push to quay.io to happen. Since this template repository provides two GitHub workflows that can build and push the image to quay.io, below are the steps to configure `IMAGE_NAME` for each of them.
+Step 2 should have provided the appropriate credentials to push the image to quay.io. This template repository provides two GitHub workflows that are configured to use these credentials to build and push the image to quay.io, but they need additonal configuration. Below are the steps to configure each of them.
```
Comment (damianavila) [inline README.md]:
```suggestion
Step 2 should have provided the appropriate credentials to push the image to quay.io. This template repository provides two GitHub workflows that are configured to use these credentials to build and push the image to quay.io, but they need additional configuration. Below are the steps to configure each of them.
```

File: README.md [hunk 4]
```diff
@@ -55,7 +110,15 @@ To enable pushing to the appropriate quay.io repository, edit line 35 of [build.
 * replace `<quay-username>/<repository-name>` with the info of the `quay.io` repository created at step 2
 * commit the changes you've made to `build.yaml`
 
-If you want to see the log of the Github Action that results you can see this via the Github Actions tab on your image repository. If you are triggering this action by merging a PR you should look at the Github Actions tab (and this will show "pushing quay.io/..."). The build log linked from the PR page is not the one for the merge, but the build done for the PR, prior to the merge (and will therefore, by default, not show "pushing quay.io/..."). 
+![IMAGE_NAME](images/image-name-in-build-workflow.png)
+
+You can checout the logs of this GitHub Workflow via the Github Actions tab on your image repository.
```
Comment (damianavila) [inline README.md]:
```suggestion
You can checkout the logs of this GitHub Workflow via the Github Actions tab on your image repository.
```

**Review by damianavila — APPROVED** [review]
Minor typos (I suggested changes). After those are fixed, LGTM.

**Comment by yuvipanda** [comment]
This is great! Can we move this to docs.2i2c.org tho? See https://github.com/2i2c-org/hub-user-image-template/issues/11

**Comment by choldgraf** [comment]
I'm also +1 on moving this to docs.2i2c.org, perhaps as a sub-page of https://docs.2i2c.org/en/latest/admin/howto/environment.html

What if we moved everything from here and downward into its own page on docs.2i2c.org: https://github.com/2i2c-org/hub-user-image-template#how-to-create-and-use-a-custom-user-image-for-your-hub

**Comment by GeorgianaElena** [comment]
Thank you all for the feedback! I've just updated the PR, by removing the `how-to` section and related images and opened https://github.com/2i2c-org/docs/pull/137 to add it there

**Review by damianavila — APPROVED** [review]

**Pull request merged**